server/src/main/java/io/deephaven/server/session/SessionState.java

```diff
@@ -678,6 +679,14 @@ public Ticket getExportId() {
             return ExportTicketHelper.wrapExportIdInTicket(exportId);
         }
 
+        /**
+         * @return the ticket bytes for this export; note if this is a non-export the returned ticket will not resolve
+         *         to anything and is considered an invalid ticket
+         */
+        public byte[] getExportIdBytes() {
```

Comment:
appears to be only called by ReferenceImpl.ticket(), which itself is test-only? is there a reason we can't just make Reference.ticket() return int, and let the test turn it into byte[]?

If seems to me that if Reference.ticket() ends up mattering in the api internals from what i know so far, it would be as a ticket, or an int could be sufficient, rather than byte[] (which is mutable).

Reply:
I was wondering about returning an int here, but also thought that at some level we wanted to not expose user-facing client apis to implementation details about exports - ie, having them remain as opaque bytes.

Reply:
But this is all internal right? SessionState itself isn't "part of the api"?

Reply:
I think what it comes down to is:

Should we allow FetchObjectResponse.data to contain raw ticket bytes? It's not 100% necessary, b/c the serialization can use Reference.index() (which will index into the FetchObjectResponse.typed_export_id array).

For example, here is how the json plugin uses Reference.index():

```json
type: deephaven.plugin.json.Node
size: 73
exportId: deephaven.plugin.json.Node:export/-1
{
  "b": "goodbye",
  "c": {
    "type": "deephaven.plugin.json.Node",
    "index": 0
  }
}
```

Should the following be "permitted"?


```json
type: deephaven.plugin.json.Node
size: 73
exportId: deephaven.plugin.json.Node:export/-1
{
  "b": "goodbye",
  "c": {
    "type": "deephaven.plugin.json.Node",
    "ticket": "<Reference.ticket() as base-16>"
  }
}
```

Reply:
Removed Reference.ticket()